api/handler/object_list.go

```diff
@@ -36,6 +36,7 @@ func (h *handler) ListObjectsV1Handler(w http.ResponseWriter, r *http.Request) {
 	if err = api.EncodeToResponse(w, encodeV1(params, list)); err != nil {
 		h.logAndSendError(w, "something went wrong", reqInfo, err)
 	}
+	api.WriteSuccessResponseHeadersOnly(w)
```

Comment:
I suppose it has no sense. Body has already sent one line 36 `api.EncodeToResponse(w, encodeV1(params, list)`

Reply:
It's strange but earlier I added this line to fix problems with s3-tests, but now they work without this line. Okay, removed.